src/coreclr/vm/riscv64/stubs.cpp

```diff
@@ -1081,96 +1081,92 @@ void StubLinkerCPU::EmitMovConstant(IntReg reg, UINT64 imm)
     }
 }
 
-void StubLinkerCPU::EmitCmpImm(IntReg reg, int imm)
+void StubLinkerCPU::EmitJumpRegister(IntReg regTarget)
 {
-    _ASSERTE(!"RISCV64: not implementation on riscv64!!!");
+    Emit32(0x00000067 | (regTarget << 15));
 }
 
-void StubLinkerCPU::EmitCmpReg(IntReg Xn, IntReg Xm)
+// Instruction types as per RISC-V Spec, Chapter 24 RV32/64G Instruction Set Listings
+static unsigned ITypeInstr(unsigned opcode, unsigned funct3, unsigned rd, unsigned rs1, int imm12)
```

Comment:
Can be moved. Since I'm fairly new to the codebase, what's a good place to put code shared between VM and JIT? `utilcode`?

There's some more redundancy between the JIT and VM emitter code, e.g. `isValid*imm*()`, that would be worth factoring out to a common place.